jetstream/tests/test_config.py

```diff
@@ -1,14 +1,132 @@
+import datetime
+from textwrap import dedent
+
 import pytest
+import toml
+from metric_config_parser.analysis import AnalysisSpec
+from metric_config_parser.config import Config, DefaultConfig, Outcome
+from metric_config_parser.outcome import OutcomeSpec
 from mozanalysis.experiment import EnrollmentsQueryType
 
-from jetstream.config import ConfigLoader, _ConfigLoader
+from jetstream.config import ConfigLoader, _ConfigLoader, validate
+from jetstream.dryrun import DryRunFailedError
 from jetstream.platform import (
     Platform,
     PlatformConfigurationException,
     _generate_platform_config,
 )
 
 
+class TestConfig:
```

Comment:
moving these test cases from integration to here because the GCP credentials used for running integration tests interfere with the GCP credentials used for the dry run function (and don't have the same permissions). The `validate()` methods call the dry run, but don't do anything in the integration GCP project, so this should be ok

Reply:
Hm, not ideal for something that requires an external service to live outside the integration tests, but I see the problem so I guess it's ok.